src/arrayMethodSort.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
+const compare = (a, b) => {
+  return a.toString() > b.toString() ? 1 : -1;
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = compare) {
+    let valueCopy = 0;
+
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          valueCopy = this[i];
+          this[i] = this[j];
+          this[j] = valueCopy;
+        }
+      }
+    }
```

Comment:
The bubble sort algorithm is correctly implemented, but it is not the most efficient sorting method for large arrays. If performance is a concern, consider implementing a more efficient algorithm like quicksort or mergesort.